actions.go

```diff
@@ -61,6 +61,31 @@ var NotImplementedExecute = func(s *discordgo.Session, m *discordgo.MessageCreat
 }
 
 var (
+	// RootAction is the first action call by Kadok for resolve
```

Comment:
this comments refer to `RootAction` on top of `StatusAction`

Reply:
Good point ! Will be corrected